src/fr/animesama/src/eu/kanade/tachiyomi/extension/fr/animesama/AnimeSama.kt

```diff
@@ -62,19 +63,52 @@ class AnimeSama : ParsedHttpSource() {
 
     // Search
     override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
-        val url = "$baseUrl/template-php/defaut/fetch.php"
+        val url = "$baseUrl/catalogue/searchbar.php"
         val formBody = FormBody.Builder()
             .add("query", query)
             .build()
 
         return POST(url, headers, formBody)
     }
-    override fun searchMangaSelector() = "a[href*='catalogue']"
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        val entries: MutableList<SManga> = mutableListOf()
+
+        val document = response.asJsoup()
+
+        document.select(searchMangaSelector()).forEach { element ->
+            val fetchExistentSubMangas = GET(element.select("a").attr("href"), headers)
+            val page = client.newCall(fetchExistentSubMangas).execute()
```

Comment:
this will result in n calls for n manga, please avoid this.

Reply:
Pagination is an option but please avoid making individual calls for each manga in browse. 
I don't follow what you mean by scanlators? You can add them to chapters of that what you are asking